legend-engine-xt-nonrelationalStore-mongodb-executionPlan-test/src/test/java/org/finos/legend/engine/plan/execution/stores/mongodb/test/Test_Pure_MongoDB_ExecutionPlan.java

```diff
@@ -0,0 +1,50 @@
+// Copyright 2023 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package org.finos.legend.engine.plan.execution.stores.mongodb.test;
+
+import junit.framework.TestSuite;
+import org.finos.legend.pure.m3.execution.test.PureTestBuilder;
+import org.finos.legend.pure.m3.execution.test.TestCollection;
+import org.finos.legend.pure.runtime.java.compiled.execution.CompiledExecutionSupport;
+import org.finos.legend.pure.runtime.java.compiled.testHelper.PureTestBuilderCompiled;
+import org.testcontainers.DockerClientFactory;
+
+public class Test_Pure_MongoDB_ExecutionPlan
+{
+    public static TestSuite suite()
+    {
+        TestSuite suite = new TestSuite();
+        if (DockerClientFactory.instance().isDockerAvailable())
+        {
+            System.setProperty("mongo_pwd", "sa");
```

Comment:
can we review this??  don't want to put this out there, can be a config that needs to be set.  We can add appropriate logging to indicate that some palceholder value is being used, (like PASSWORD_TO_CHANGE)

Reply:
Looking at PureTestHelper they seem to do this in many other places for tests, we can certainly add this to a config. Elastic equivalent uses the same pattern as well 

we can try to do something else if we prefer but would need to inject at some point